### PR TITLE
change readme file

### PR DESCRIPTION
**Pull request by h-ping**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,55 @@
...
+    <a href="https://discord.gg/3HYaR6USM7"><img src="https://img.shields.io/discord/864829036294307881?color=5865F2&logo=discord&logoColor=white&label=discord" alt="Discord server" /></a>
+    <a href="https://t.me/joinchat/4-ocuURAr2thODFh"><img src="https://img.shields.io/badge/telegram-blue?color=blue&logo=telegram&logoColor=white" alt="Telegram group" /></a>
+    <a href="https://www.youtube.com/c/FootprintAnalytics"><img alt="YouTube Channel Subscribers" src="https://img.shields.io/youtube/channel/subscribers/UCKwZbKyuhWveetGhZcNtSTg?style=social"></a>
+  </p>
+</div>
 
-[Metabase](https://www.metabase.com) is the easy, open-source way for everyone in your company to ask questions and learn from data.
+## Introduce
+Footprint Analytics is building blockchain's most comprehensive data infrastructure with tools to help developers, analysts, and investors get unrivaled real-time GameFi, DeFi, and NFT insights.
+The engine indexes, cleans and abstracts data from 24+ chains and counting — letting users build charts and dashboards without code using a drag-and-drop interface as well as with SQL or Python. [Explore Web3 data](https://www.footprint.network/dashboards)
```
Comment (navyxie) [inline README.md:23]: This description is a bit old, you can remove python

Comment (navyxie) [reply README.md:23]: This picture can be replaced by the covered chain, and the marketplace

File: README.md [hunk 2]
```diff
@@ -1,67 +1,55 @@
...
+Footprint Analytics also provides a unified data API for NFTs, GameFi, and DeFi across all major chain ecosystems to build your applications fast.
```
Comment (navyxie) [inline README.md:24]: To add some API use cases, scene description

File: README.md [hunk 3]
```diff
@@ -1,67 +1,55 @@
...
 
-## Supported databases
+![footprint API snapshot](https://static.footprint.network/github/footprint_api_readme.png)
 
-- [Officially supported databases](./docs/databases/connecting.md#connecting-to-supported-databases)
-- [Partner and Community drivers](./docs/developers-guide/partner-and-community-drivers.md)
 
-## Installation
+## Footprint Data
+As of Jan 2023, we parse more data from 22 different chains than any other platforms. The Footprint Analytics database automatically picks up blocks, logs, traces, and transactions on the blockchain. It supplements this with community-contributed data and data from 3rd party APIs (e.g. token price data from Coingecko.) All this data is originally raw and unstructured. We structure it to fit into categories, e.g. borrowing, lending, yield farming, etc. This way, any data from the blockchain is easily accessible by anyone.
```
Comment (navyxie) [inline README.md:46]: 24 chains

File: README.md [hunk 4]
```diff
@@ -1,67 +1,55 @@
...
 
-Metabase can be run just about anywhere. Check out our [Installation Guides](https://www.metabase.com/docs/latest/operations-guide/installing-metabase.html).
+Our core competitive advantage is our Footprint Analytics Platform, powered by the Footprint Machine Learning Platform. It turns the Bronze Data into Silver, then Gold using some technical means of data ETL, such as Python and SQL. In the future, we plan to make the ETL code, including the code from Bronze to Silver parsing, open source.
 
-## Contributing
+We also enable any organization to tap into this trove of structured data with our blockchain data API.
```
Comment (navyxie) [inline README.md:50]: Add some examples of data design, such as NFT domain, refer to https://docs.footprint.network/docs/domain

**Comment by ydpzg** [comment]
the content is merged

**Pull request closed without merging**